marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.module.scss

```diff
@@ -76,6 +83,18 @@ span.readOnlyLabel {
     margin-bottom: 8px !important;
 }
 
+.formItemTgtCollections { 
```

Comment:
I mentioned this in another PR - I always recommend spelling out words, with the possible exception of local variables that have a very small scope. More verbose names are also useful when searching for references to functions/classes/etc, as you get fewer false positives. So let's not do "src" and "tgt" - let's use "source" and "target", the same words we normally use (using these abbreviations also raises the question of why we abbreviate some words but not others - like if we're doing src/tgt, why not do colls instead of collections? Not abbreviating means we don't have to think through that or explain it).

Reply:
Done